Library/Homebrew/test/cask/system_command_spec.rb

```
@@ -136,4 +197,11 @@
       }.to be_a_success
     end
   end
+
+  describe "given an invalid variable name" do
+    it "returns an ArgumentError" do
```

“raises” instead of ”returns”